product_label_image/reports/product_template_report.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="paperformat_label_layout" model="report.paperformat">
+        <field name="name">Label Sheet</field>
+        <field name="default" eval="True" />
+        <field name="format">A6</field>
+        <field name="orientation">Landscape</field>
+        <field name="margin_top">29</field>
+        <field name="margin_bottom">10</field>
+        <field name="margin_left">4</field>
+        <field name="margin_right">4</field>
+    </record>
+
+    <template id="report_simple_label_image_4x6">
+        <tr>
+            <td t-att-style="make_invisible and 'visibility:hidden;'">
+                <div style="width:100%;margin-bottom:200px;" class="row">
+
+                    <div
+            style="width:310px;height:210px; border:2px solid #555555 !important;padding: 2px 4px;"
+          >
+                        <div
+              style="height:86px;margin-bottom: 18px;overflow:hidden;background-color: ghostwhite;"
+            >
+                            <strong
+                t-field="product.display_name"
+                style="font-size: 16px;"
+              />
+                        </div>
+
+                        <div class="text-center">
+                            <t t-if="barcode">
+                                <div
+                  t-out="barcode"
+                  style="padding:5px;"
+                  t-options="{'widget': 'barcode', 'symbology': 'auto', 'img_style': 'width:71mm;height:15mm;'}"
+                />
+                                <span class="text-center" t-out="barcode" />
+                            </t>
+                        </div>
+                    </div>
+                    <t t-if="product.image_512">
+                        <div
+              style="width:310px;height:210px; border:2px solid #555555;border-left:1px;"
+            >
```

Comment:
To me it seems as if your code formatter does also make the code look very weird in cases there is no fixed technical ruleset regarding how it should look like.

Reply:
Its auto formatted based on pre-commit files given in the repo.

Reply:
@manengel Code auto formatted according to pre-commit files given in the repo. If we not follow it, PR failed. So we must need to follow it.